ui/admin/app/components/form/storage-bucket/minio/index.hbs

```diff
@@ -0,0 +1,341 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+  {{! name }}
+  <Hds::Form::TextInput::Field
+    name='name'
+    @isOptional={{true}}
+    @value={{@model.name}}
+    @isInvalid={{@model.errors.name}}
+    @type='text'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'name')}}
+    as |F|
+  >
+    <F.Label>{{t 'form.name.label'}}</F.Label>
+    <F.HelperText>{{t 'form.name.help'}}</F.HelperText>
+    {{#if @model.errors.name}}
+      <F.Error as |E|>
+        {{#each @model.errors.name as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  {{! description }}
+  <Hds::Form::Textarea::Field
+    name='description'
+    @isOptional={{true}}
+    @value={{@model.description}}
+    @isInvalid={{@model.errors.description}}
+    disabled={{form.disabled}}
+    as |F|
+  >
+    <F.Label>{{t 'form.description.label'}}</F.Label>
+    <F.HelperText>{{t 'form.description.help'}}</F.HelperText>
+    {{#if @model.errors.description}}
+      <F.Error as |E|>
+        {{#each @model.errors.description as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::Textarea::Field>
+
+  {{! scope }}
+  {{#if @model.isNew}}
+    <Hds::Form::Select::Field
```

Comment:
It seems like the scope dropdown gets reset when you switch between the aws and minio type in the form but it's still there in the data model

Reply:
@ZedLi 
Thanks for the catch. Yeah I am trying to review this part, I do believe, as per current implementation is a bit of a mess for such simple field: tracked properties, action that calls another internal method... I do believe this could be based on model value solely without need for those mentioned methods (`updateScopeFieldDescription`, `updateScope` and `scopeFieldDescription`) I will keep you posted when I push the updates.

Reply:
@ZedLi 
Once again, thanks for the catch.

I changed the behaviour of how we set the initial scope and how this component behaves.

**Initial scope:**
Previous implementation: sets scope to global at the new route file. When changing provider the route gets refreshed because we use queryParams, therefore scope is set to global overriding user previously selected scope.

Current implementation: we do not set the scope until the user specified.

**Component behaviour**
With the initial scope changes, now we know if @model.scope is not set, means the user did not choose yet. So we do not need a tracked property to choose what helper text to show (scopeFieldDescription) neither a method to update the description (updateScopeFieldDescription).